src/sentry/api/serializers/models/sentry_app_component.py

```diff
@@ -15,3 +15,16 @@ def serialize(self, obj, attrs, user):
                 "name": obj.sentry_app.name,
             },
         }
+
+
+class SentryAppAlertRuleActionSerializer(Serializer):
+    def serialize(self, obj, attrs, user, **kwargs):
+        return {
+            "id": f"sentry.sentryapp.{obj.sentry_app.slug}",
+            "uuid": str(obj.uuid),
+            "actionType": "sentryapp",
+            "prompt": f"{obj.sentry_app.name}",
+            "enabled": True,
+            "label": obj.schema.get("title", ""),
```

Comment:
nit: What's the difference between prompt and label? Clearer names might help, where are they going to appear?

Reply:
@leeandher `prompt` is what gets rendered in the dropdown. `label` is what gets rendered once selected. This response body is setup to be inline with our current response. Probs not going to change the names in this PR.